src/main/java/com/google/api/codegen/util/LicenseHeaderUtil.java

```diff
@@ -15,88 +15,35 @@
 package com.google.api.codegen.util;
 
 import com.google.api.codegen.ConfigProto;
-import com.google.api.codegen.LanguageSettingsProto;
-import com.google.api.codegen.LicenseHeaderProto;
-import com.google.api.tools.framework.model.Diag;
-import com.google.api.tools.framework.model.DiagCollector;
-import com.google.api.tools.framework.model.SimpleLocation;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Charsets;
-import com.google.common.base.Preconditions;
-import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableList;
 import com.google.common.io.CharStreams;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
-import javax.annotation.Nullable;
 
 public class LicenseHeaderUtil {
-  public static final String DEFAULT_LICENSE_FILE = "license-header-apache-2.0.txt";
-  public static final String DEFAULT_COPYRIGHT_FILE = "copyright-google.txt";
+  @VisibleForTesting static final String DEFAULT_LICENSE_FILE = "license-header-apache-2.0.txt";
+  static final String DEFAULT_COPYRIGHT_FILE = "copyright-google.txt";
```

Comment:
Should both constants be annotated? And, should the annotation be on the line above?

Reply:
will annotate both! google-java-format put the annotation in front instead of above